### PR TITLE
cosmetic doc improvements part 4

### PR DESCRIPTION
**Pull request by NathanQingyangXu**
final part. feel free to change.

### CODE REVIEW

**Review by artembilan — CHANGES_REQUESTED** [review]
Just a couple questions.
Thanks

File: spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/receiving-messages/sequencing.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [[sequencing]]
-= Starting `@KafkaListener`+++s+++ in Sequence
+= Starting `@KafkaListener`s in Sequence
```
[the remark this answers is not in the document]
Comment (NathanQingyangXu) [reply spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/receiving-messages/sequencing.adoc:2]: a mistake. updated. thx

File: spring-kafka-docs/src/main/antora/modules/ROOT/pages/retrytopic/topic-naming.adoc [hunk 2]
```diff
@@ -5,26 +5,26 @@ Retry topics and DLT are named by suffixing the main topic with a provided or de
 
 Examples:
 
-"my-topic" -> "my-topic-retry-0", "my-topic-retry-1", ..., "my-topic-dlt"
+"my-topic" +++->+++ "my-topic-retry-0", "my-topic-retry-1", ..., "my-topic-dlt"
```
[the remark this answers is not in the document]
Comment (NathanQingyangXu) [reply spring-kafka-docs/src/main/antora/modules/ROOT/pages/retrytopic/topic-naming.adoc:8]: I thought the special arrow rending seems not intended and the "raw" text seems more desirable. Reverted back.

[the remark this answers is not in the document]
Comment (NathanQingyangXu) [reply spring-kafka-docs/src/main/antora/modules/ROOT/pages/retrytopic/topic-naming.adoc]: I also reverted back a similar change commited previously: https://github.com/spring-projects/spring-kafka/pull/2879/commits/83d1eafcb5d07f5b9bcf3127ec2a23f9440de170

**Pull request merged**